dotCMS/src/main/java/com/dotmarketing/util/DateUtil.java

```diff
@@ -74,6 +75,8 @@
 
 	public static final String UTC = "UTC";
 
+	public static final SimpleDateFormat EXPORTING_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd_hh-mm-ss");

```

Comment:
shouldn't we use Config for the pattern?

Reply:
It's kind of opinionated since it is for filename generation.